utils/http_test.go

```diff
@@ -51,6 +51,7 @@ func TestRootHandlerError(t *testing.T) {
 	defer ts.Close()
 
 	res, err := http.Get(ts.URL)
+	require.NoError(t, err)
 	if res.StatusCode != http.StatusInternalServerError {
 		t.Fatalf("Expected 500, received %d", res.StatusCode)
```

Comment:
I can if you'd like, though it isn't as useful to use the constant here because this is for a test error message instead of an actual data key or comparison value.

I'm leaning to leave it as is because I think it's more readable, otherwise it'll look like:
`t.Fatalf("Expected %d, received %d", http.StatusInternalServerError, res.StatusCode)`